.github/workflows/continous-integration.yaml

```diff
@@ -0,0 +1,32 @@
+name: Continous Integration
+
+# This workflow is triggered on pushes to the repository.
+on: [push, pull_request]
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        os:
+          - windows-latest
+          - ubuntu-latest
+          - macos-latest
+        node-version: [10, 12, 13]
+
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Update NPM
+      run: npm i -g npm
+    - name: Bootstrap project
+      run: npm ci --ignore-scripts
+      env:
+        CI: true
+    - name: Build project
+      run: npm run build
+    - name: Run tests
+      run: npm run test:ci
```

Comment:
Please add jobs for linting our code, commit messages and verifying the doc pages. These jobs should be executed on a single platform+node-version and can be run in parallel with other test tasks.

https://github.com/strongloop/loopback-next/blob/947eb10d311d10d9a0b0f6c17dbc8e373df0a6f9/.travis.yml#L28-L46

We also need to run acceptance tests, see here:

https://github.com/strongloop/loopback-next/blob/947eb10d311d10d9a0b0f6c17dbc8e373df0a6f9/.travis.yml#L47-L105